cart/views.py

```diff
@@ -13,8 +16,8 @@ class CartView(APIView):
     authentication_classes = [authentication.TokenAuthentication]
```

Comment:
Please explain why we are using permission_classes IsAuthenticated and authentication_classes together .

Reply:
fixed